apps/files_sharing/lib/Notification/Notifier.php

```diff
@@ -192,6 +192,9 @@ protected function parseShareInvitation(IShare $share, INotification $notificati
 				}
 
 				$group = $this->groupManager->get($share->getSharedWith());
+				if ($group === null) {
+					throw new \InvalidArgumentException('Temporary failure');
```

Comment:
I vote for:
```suggestion
					throw new AlreadyProcessedException();
```

which deletes the notification as well so we never check it again?

Reply:
Sounds good. I wasn't sure so thanks for confirming :-)